vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -213,6 +213,10 @@ public String toString() {
     @SerializedName("spatial")
     public Spatial mSpatial;
 
+    @Nullable
+    @SerializedName("live")
+    private Live mLive;
```

Comment:
Everything else is public, might as well stay consistent.  FWIW, I have a branch open that makes all fields private: https://github.com/vimeo/vimeo-networking-java/tree/make-fields-private however there are a couple issues with Stag that are preventing me from opening up a PR for this.